sources/academy/webscraping/scraping_basics_python/04_downloading_html.md

```diff
@@ -142,10 +142,10 @@ Letting our program visibly crash on error is enough for our purposes. Now, let'
 
 ### Scrape Amazon
 
-Download HTML of a product listing page, but this time from a real world e-commerce website. For example this page with Amazon search results:
+Download HTML of a product listing page, but this time from a real world e-commerce website. For example this page with AliExpress search results:
```

Comment:
I decided to do this because this is like... the very first exercise, and I don't want the reader to end up with an error. I don't want to direct them somewhere else when they're touching the internet for the first time with their tiny program. So hoping for better results with Ali, which is a real world e-commerce website as well, but doesn't seem to give so much damn about scraping.